tests/unit/test_kernel.py

```diff
@@ -951,7 +951,7 @@ def test_scaled_laplacian_kernel_gradients_wrt_x(self) -> None:
         .. math:
 
             - \text{output_scale}\operatorname{sgn}{(x - y)}{2length\_scale^{2}}e^{-\frac{\lVert x - y\rVert_1}{2 \text{length_scale}^2}}
-        """
+        """  # noqa: E501
```

Comment:
Line too long? Is there a way we can rewrite the LaTeX so that isn't a problem?

Reply:
Yes. In the CONTRIBUTING.md we ask "Prioritise overfull lines for mathematical expressions over artificially splitting them into multiple equations in both comments and docstrings." so I think this should be left as is?

Reply:
Fair point!